src/components/App.js

```diff
@@ -87,4 +88,11 @@ const Guard = ({ redirectTo, condition, children }) => {
   }
 }
 
+App.propTypes = {
+  currentUser: PropTypes.shape({
```

Comment:
This also appears to be referring to state, not props. `App` should generally not have props.